tests/unit/test_config.py

```diff
@@ -47,8 +47,9 @@
 
 from cylc.flow.cycling.iso8601 import ISO8601Point
 
-
-Fixture = Any
+if TYPE_CHECKING:
+    from pathlib import Path
+    Fixture = Any
```

Comment:
FYI this is not flagged by flake8 - we ignore the TC rules as these are tests not production code